website/templates/public/login.mako

```diff
@@ -80,176 +86,198 @@
                 </div>
             </div>
             <div class="form-group">
-                <label for="inputPassword3" class="col-sm-3 control-label">Password</label>
-                    <div class="col-sm-9">
-                    <input
-                        type="password"
-                        class="form-control"
-                        id="inputPassword3"
-                        placeholder="Password"
-                        data-bind="value: password"
-                        name="password"
-                    >
+                <label for="inputPassword3" class="col-sm-2 control-label">Password</label>
+                    <div class="col-sm-10">
+                        <input
+                            ${'autofocus' if existing_user else ''}
+                            type="password"
+                            class="form-control"
+                            id="inputPassword3"
+                            placeholder="Password"
+                            data-bind="value: password"
+                            name="password"
+                        >
                 </div>
             </div>
-            <div class="form-group">
-                <div class="col-sm-offset-3 col-sm-9">
-                    <div class="checkbox">
-                    <label><input type="checkbox"> Remember me</label>
+            %if existing_user:
+                <div class="col-sm-12 m-t-sm">
+                <div class="form-group col-sm-7 remember-me-checkbox">
```

Comment:
The grid CSS here looks off. I don't think `col-*` classes are meant to be used on the same element as `form-group`. Also, each row needs to have 12 columns. Can you look into fixing this up?

Using the grid properly will hopefully allow you to remove one-off classes like `remember-me-checkbox`.